src/assets/scripts/client/navigationLibrary/FixCollection.js

```diff
@@ -127,13 +148,7 @@ class FixCollection extends BaseCollection {
      * @return {array<FixModel>}
      */
     findRealFixes() {
-        const realFixList = _map(this._items, (item) => {
-            if (item.name.indexOf('_') !== 0) {
-                return item;
-            }
-        });
-
-        return _compact(realFixList);
+        return this._items.filter((fix) => fix.isRealFix);
```

Comment:
😌

Reply:
I had to google that emoji to make sure it was a good thing...